### PR TITLE
Added explorer integration doc

### PR DESCRIPTION
**Pull request by alexey708**
Added explorer integration doc in docs/families/btc/integration_14_explorer.md

### CODE REVIEW

File: docs/families/btc/integration_14_explorer.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Change Blockchain Explorer Params {#integration_14_explorer}
+
+[TOC]
+
+# Overview
+
+Clone https://github.com/VeriBlock/vbk-ri-btc-explorer repo with explorer source code.
+
+Create json file at `app/coins/youraltcain.js` (you should get other files as example). This file contain params of your altchain. Fill that file with genesis, chain and other params of your altchain. For example to get info from genesis coinbase TX you can print HEX encoded TX data to log in genesis block creation function (`CreateGenesisBlock`) and then use RPC `decoderawtransaction` to get info from this TX.
```
Comment (blackyblack) [inline docs/families/btc/integration_14_explorer.md:9]: youraltcoin.js

Comment (blackyblack) [inline docs/families/btc/integration_14_explorer.md:9]: This file contains params

Comment (blackyblack) [reply docs/families/btc/integration_14_explorer.md:9]: "For example to get info from genesis coinbase TX you can print HEX encoded TX data" - do we have any better way of configuring coin params?

**Comment by Warchant** [comment]
Please create new branch with these changes from last develop branch.

**Pull request closed without merging**